src/pages/Home.js

```diff
@@ -0,0 +1,60 @@
+import * as api from 'api'
+
+import { EmptyListView, LightningTalkCard } from 'components'
+import React, { useEffect, useState } from 'react'
+
+import { isEmpty } from 'lodash'
+
+export default function Home() {
+  const [isLoading, setIsLoading] = useState(true)
+  const [data, setData] = useState([])
+
+  useEffect(() => {
+    getLightningTalksPolls()
+  }, [])
+
+  async function getLightningTalksPolls() {
+    try {
+      setIsLoading(true)
+      const response = await api.listSortedLightningTalks()
+
+      if (response === false) throw new Error('API Error')
+
+      setData(response)
+      setIsLoading(false)
+    } catch (error) {
+      setIsLoading(false)
```

Comment:
If api fetch has an error, the user won't see an issue, just `No lightning talks polls have been created yet. Be the first to publish the first one 😃` without a way to fix it.

Reply:
You're right. I did so as a compromise. On one hand, to fulfill the test requirement, which asks to show a message encouraging the users to submit a new lightning talk in case of an empty list.

On another, having a message vague and generic.

A better compromise would have been to use two messages, one for the empty list, and one for API errors.

A perfect solution would be parsing every API error (corrupt input, authorization,  etc.) and providing the user with the proper messaging and action to take.